packages/client/src/schedule-helpers.ts

```diff
@@ -312,6 +322,8 @@ export async function decodeScheduleAction(
   pb: temporal.api.schedule.v1.IScheduleAction
 ): Promise<ScheduleDescriptionAction> {
   if (pb.startWorkflow) {
+    const jsonConverter = new JsonPayloadConverter();
```

Comment:
Use the data converters configured on the client (i.e. custom payload converter and codec).

Reply:
yup, done